.devcontainer/development/Dockerfile

```
@@ -72,6 +72,10 @@ RUN npm install -g @google/gemini-cli \
 RUN npm install -g @github/copilot && \
     echo "✅ GitHub Copilot CLI installed: $(copilot --version)"
 
+# Install OpenAI Codex CLI (standalone coding agent)
+RUN npm install -g @openai/codex && \
```

Hadolint (2.14.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)